netCDFConfig.cmake.in

```diff
@@ -40,3 +40,7 @@ set(netCDF_HAS_DAP4 @HAS_DAP4@)
 set(netCDF_HAS_DISKLESS @HAS_DISKLESS@)
 set(netCDF_HAS_MMAP @HAS_MMAP@)
 set(netCDF_HAS_JNA @HAS_JNA@)
+
+if (netCDF_HAS_HDF4 OR netCDF_HAS_HDF5)
+  find_dependency(HDF5)
+endif ()
```

Comment:
To use the `find_dependency` macro, you need to include `CMakeFindDependencyMacro`.

```suggestion
include(CMakeFindDependencyMacro)
if (netCDF_HAS_HDF4 OR netCDF_HAS_HDF5)
  find_dependency(HDF5)
endif ()
```

Reply:
Whoops, yep. Thanks. I added it inside the `if` block though.

Reply:
That's fine with me.

Can you update this in the ParaView superbuild? That's where I ran into this problem.